### PR TITLE
Add FXIOS-8270 [Onboarding Customization] Add telemetry

### PR DESCRIPTION
**Pull request by adudenamedruby**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-8270)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/18358)

## :bulb: Description
Adds telemetry for multiple choice buttons.

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [x] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [x] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

**Comment by adudenamedruby** [comment]
@mergifyio backport release/v125

File: firefox-ios/Client/Frontend/Onboarding/Models/OnboardingTelemetryUtility.swift [hunk 1]
```diff
@@ -55,12 +70,13 @@ class OnboardingTelemetryUtility: OnboardingTelemetryProtocol {
         using cardName: String
     ) -> [String: String] {
         typealias Key = TelemetryWrapper.EventExtraKey
-        return [
+        let thing = [
```
[the remark this answers is not in the document]
Comment (adudenamedruby) [reply firefox-ios/Client/Frontend/Onboarding/Models/OnboardingTelemetryUtility.swift:73]: This was from testing and I forgot it in there. Shame on me!

**Comment by adudenamedruby** [comment]
@dnarcese 
# Request for data collection review form

1) What questions will you answer with this data?

* Do users tap on these customization options if they're shown?

2) Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements?

Product and business requirements, as part of the Onboarding Customization experiment.

3) What alternative methods did you consider to answer these questions? Why were they not sufficient?

This data isn't currently collected in Firefox for iOS, and there are no alternatives.

4) Can current instrumentation answer these questions?

No.

5) List all proposed measurements and indicate the category of data collection for each measurement, using the [Firefox data collection categories](https://wiki.mozilla.org/Data_Collection) found on the Mozilla wiki.

This PR adds the following measurements to the the Glean `onboarding` ping:

| Measurement             | Description                                                                                                                                                                                        | Data Collection Category | Tracking Bug # |
|-------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|----------------|
| `multiple-choice-button-tap`  | Adding `theme-dark`, `theme-light`, `theme-sytem`, `toolbar-top`, `toolbar-bottom`, as a possible value.                                                                                                                                               | 2                        | 19678          |

6) Please provide a link to the documentation for this data collection which describes the ultimate data set in a public, complete, and accurate way.

This collection is Glean so is documented [in the Glean Dictionary](https://dictionary.telemetry.mozilla.org).

7) How long will this data be collected?  Choose one of the following:

This data will be collected for a period of 6-months.

8) What populations will you measure?

All populations.

9) If this data collection is default on, what is the opt-out mechanism for users?

These collections are Glean. The opt-out can be found in Settings.

10) Please provide a general description of how you will analyze this data.

General reporting on Looker and Redash, as well as experiment-specific analyses.

11) Where do you intend to share the results of your analysis?

Internal analyses will be shared with the PM team in Looker.

12) Is there a third-party tool (i.e. not Glean or Telemetry) that you are proposing to use for this data collection?

No.

**Comment by dnarcese** [comment]
r+
# Data Review Form

_1) Is there or will there be **documentation** that describes the schema for the ultimate data set in a public, complete, and accurate way?_ 

Yes, telemetry is documented at https://dictionary.telemetry.mozilla.org/apps/firefox_ios

_2) Is there a control mechanism that allows the user to turn the data collection on and off?_ 

Yes, Firefox iOS provides data controls in the app settings

_3) If the request is for permanent data collection, is there someone who will monitor the data over time?_

This will be reviewed by me every 6 months

_4) Using the **[category system of data types](https://wiki.mozilla.org/Firefox/Data_Collection)** on the Mozilla wiki, what collection type of data do the requested measurements fall under?_

Type 2

_5) Is the data collection request for default-on or default-off?_

Default on

_6) Does the instrumentation include the addition of **any *new* identifiers** (whether anonymous or otherwise; e.g., username, random IDs, etc.  See the appendix for more details)?_

No

_7) Is the data collection covered by the existing Firefox privacy notice?_ 

Yes

_8) Does the data collection use a third-party collection tool?_ 

No

**Pull request merged**